docker/Dockerfile

```diff
@@ -429,6 +429,17 @@ RUN --mount=type=cache,target=/root/.cache/uv \
     uv pip install --system -r requirements/build.txt \
         --extra-index-url ${PYTORCH_CUDA_INDEX_BASE_URL}/cu$(echo $CUDA_VERSION | cut -d. -f1,2 | tr -d '.')
 
+# Install EP kernels(pplx-kernels and DeepEP), NixL and DeepGEMM
+COPY tools/ep_kernels/install_python_libraries.sh install_python_libraries.sh
+COPY tools/install_nixl.sh install_nixl.sh
+COPY tools/install_deepgemm.sh install_deepgemm.sh
+ENV CUDA_HOME=/usr/local/cuda
+RUN bash install_python_libraries.sh \
+    && bash install_deepgemm.sh \
+    && bash install_nixl.sh --force \
+    # remove libraries after installation
+    && rm -rf ep_kernels_workspace deepgemm_workspace nixl_installer
```

Comment:
I thought we need these workspace because the required `so` files and others are built in there.